core/src/main/scala/kafka/zookeeper/ZooKeeperClient.scala

```diff
@@ -396,6 +396,13 @@ sealed trait AsyncResponse {
   /** Return None if the result code is OK and KeeperException otherwise. */
   def resultException: Option[KeeperException] =
```

Comment:
Is this used anywhere now?

Reply:
Thanks for the review. Yes this method is called in KafkaZkClient and Controller classes (KafkaController, PartitionStateMachine, ReplicaStateMachine).

Updated the PR.